libqrexec/libqrexec-utils.h

```diff
@@ -162,6 +162,11 @@ void buffer_append(struct buffer *b, const char *data, int len);
 void buffer_remove(struct buffer *b, int len);
 int buffer_len(struct buffer *b);
 void *buffer_data(struct buffer *b);
+/* Open /dev/null and keep it from being closed before the exec func is called.
```

Comment:
Isn't it simpler (and safer) to simply open /dev/null just before dup-ing it over 0,1,2 (in the child process already)?

Reply:
It’s definitely simpler, but I didn’t want the extra call to `open()`.  That said, giving each child process the same open file description is less than great, as it introduces shared state that should not be there.  I don’t know if `/dev/null` has any such state, but still it isn’t great, so I went ahead and switched to your approach.